pkg/types/aws/validation/platform.go

```diff
@@ -74,13 +78,13 @@ func validateUserTags(tags map[string]string, propagatingTags bool, fldPath *fie
 }
 
 // validateTag checks the following things to ensure that the tag is acceptable as an additional tag.
-// * The key and value contain only valid characters.
-// * The key is not empty and at most 128 characters.
-// * The value is not empty and at most 256 characters. Note that, while many AWS services accept empty tag values,
-//   the additional tags may be applied to resources in services that do not accept empty tag values. Consequently,
-//   OpenShift cannot accept empty tag values.
-// * The key is not in the kubernetes.io namespace.
-// * The key is not in the openshift.io namespace.
+//   - The key and value contain only valid characters.
+//   - The key is not empty and at most 128 characters.
+//   - The value is not empty and at most 256 characters. Note that, while many AWS services accept empty tag values,
+//     the additional tags may be applied to resources in services that do not accept empty tag values. Consequently,
+//     OpenShift cannot accept empty tag values.
+//   - The key is not in the kubernetes.io namespace.
+//   - The key is not in the openshift.io namespace.
```

Comment:
looks like this reformatting was applied by my IDE. looks fine/harmless to me, but will fix it if the tests complain

Reply:
It would be nice if we did not merge these changes though.